config.md

```diff
@@ -308,15 +308,23 @@ The semantics are the same as `Path`, `Args` and `Env` in [golang Cmd](https://g
 
 This OPTIONAL property contains arbitrary metadata for the container.
 This information MAY be structured or unstructured.
-Annotations are key-value maps.
+Annotations MUST be key-value maps where both the key and value are strings.
```

Comment:
On Thu, Jul 07, 2016 at 11:05:09AM -0700, Vish Kannan wrote:
> > @@ -308,15 +308,23 @@ The semantics are the same as `Path`, `Args` and `Env` in [golang Cmd](https://g
> >  
> >  This OPTIONAL property contains arbitrary metadata for the container.
> >  This information MAY be structured or unstructured.
> > -Annotations are key-value maps.
> > +Annotations MUST be key-value maps where both the key and value are strings.
> 
> are `values` optional? If yes, mention that explicitly. 

Probably best to be explicit either way.  The Go type uses
map[string]string [1], not map[string]*string, so I expect the
intention was to require a string value and not support null values
(although I'm personally a fan of not typing the content at all [2],
so I may not be the best judge of the typing intentions).

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc1/specs-go/config.go#L21-L22
[2]: https://github.com/opencontainers/runtime-spec/pull/331#issuecomment-192477931